### PR TITLE
{Service Fabric} Add securityType and SecureBootEnabled properties to Nodetype

### PR DESCRIPTION
**Pull request by mwesigwaguma**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by yanzhudd** [comment]
Please resolve the CI issues.
Please note that we will freeze the code on 06/25/2024 10:00 UTC for the upcoming release. If you want to catch the release train, please resolve the comments as soon as possible, otherwise we have to postpone this PR to the next sprint (08-06).

File: src/azure-cli/azure/cli/command_modules/servicefabric/_params.py [hunk 1]
```diff
@@ -320,16 +321,18 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
...
+        c.argument('vm_image_sku', help='The SKU of the Azure Virtual Machines Marketplace image.')
         c.argument('vm_image_version', help='The version of the Azure Virtual Machines Marketplace image. ', default='latest')
         c.argument('capacity', arg_type=capacity)
         c.argument('placement_property', arg_type=placement_property)
         c.argument('is_stateless', arg_type=get_three_state_flag(), help='Indicates if the node type can only host Stateless workloads.', default=False)
         c.argument('multiple_placement_groups', options_list=['--multiple-placement-groups', '--multi-place-groups'], arg_type=get_three_state_flag(),
                    help='Indicates if scale set associated with the node type can be composed of multiple placement groups.', default=False)
+        c.argument('security_type', arg_type=get_enum_type(SecurityType), help='Specifies the security type of the nodeType. Only TrustedLaunch is currently supported.')
+        c.argument('secure_boot_enabled', arg_type=get_three_state_flag(), help='Specifies whether secure boot should be enabled on the nodeType. Can only be used with TrustedLaunch SecurityType.')
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/servicefabric/_params.py:335]: Please use the first-person voice in the help message
```suggestion
        c.argument('security_type', arg_type=get_enum_type(SecurityType), help='Specify the security type of the nodeType. Only TrustedLaunch is currently supported.')
        c.argument('secure_boot_enabled', arg_type=get_three_state_flag(), help='Specify whether secure boot should be enabled on the nodeType. Can only be used with TrustedLaunch SecurityType.')
```

**Comment by zhoxing-ms** [comment]
@mwesigwaguma Since we haven't received a response to this PR for a long time, I will put it in the backlog first. After you have resolved its comments, we will consider adding it to the release milestone

**Pull request closed without merging**